src/components/shared/IdleSearchField.css

```diff
@@ -23,12 +23,6 @@
   background-size: 11px 11px;
 }
 
-@media(-moz-mac-yosemite-theme) {
-  .idleSearchFieldInput:focus {
-    box-shadow: 0 0 0 3px -moz-mac-focusring, inset 0 0 0.5px rgba(0,0,0,0.15), 0 0 0.5px rgba(0,0,0,0.1);
-  }
-}
-
```

Comment:
This has been removed in Firefox 58, see https://developer.mozilla.org/en-US/Firefox/Releases/58#CSS_2